framework/include/cppmicroservices/ListenerFunctors.h

```diff
@@ -151,55 +151,16 @@ namespace cppmicroservices {
    */
   typedef std::function<void(const FrameworkEvent&)> FrameworkListener;
 
-  /**
-   * \ingroup MicroServices
-   * \ingroup gr_listeners
-   *
-   * A convenience function that binds the member function <code>callback</code> of
-   * an object of type <code>R</code> and returns a <code>ServiceListener</code> object.
-   * This object can then be passed into <code>AddServiceListener()</code>.
-   *
-   * @tparam R The type containing the member function.
-   * @param receiver The object of type R.
-   * @param callback The member function pointer.
-   * @returns a ServiceListener object.
-   */
   template<class R>
-  ServiceListener ServiceListenerMemberFunctor(R* receiver, void (R::*callback)(const ServiceEvent&))
+  US_DEPRECATED ServiceListener ServiceListenerMemberFunctor(R* receiver, void (R::*callback)(const ServiceEvent&))
```

Comment:
Thanks, but can we get the documentation back :-) You invested some time in it already and it would be good to have it. Just add a short "deprecated" note there as well.